packages/amplify-category-geo/src/provider-controllers/deviceLocationTracking.ts

```diff
@@ -17,8 +17,8 @@ export const addDeviceLocationTrackingResource = async (
     providerContext: setProviderContext(context, ServiceName.DeviceLocationTracking),
   };
   // populate the parameters for the resource
-  await createDeviceLocationTrackingWalkthrough(context, trackingParams);
-  const resourceName = await addDeviceLocationTrackingWithParams(context, trackingParams);
+  const newParams = await createDeviceLocationTrackingWalkthrough(context, trackingParams);
```

Comment:
either make trackingParams `var`/`let` or rename `newParams` to something meaningful please.